requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy==1.22.4
 cython==0.29.32
 cgen==2020.1
+pyqir==0.8.2
```

Comment:
Should you add `pyqir` to [setup.py](https://github.com/dwavesystems/dwave-gate/blob/44970734dff605d72b5d7b940a319abb5da8f07f/setup.py#L51), or you want dwave-gate working without it?